src/components/atomic/Alert.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div :class="getClass" @click="close">{{ msg }}</div>
```

Comment:
The alert div should get `role="alert"` for accessibility. (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role)

Reply:
added role with cfbe90f